### ISSUE TITLE
Issue of auto-deployment on Heroku

### ISSUE THREAD

**Issue by meets2tarun** [fossasia/meilix-generator#88]
**I'm submitting a ...**  
- [x] bug report

**Actual Behaviour**
We are getting this as an error and as @xeon-zolt mention, it asks for credit card credentials also but here it's something else:
![screenshot from 2017-07-25 21-25-06](https://user-images.githubusercontent.com/15213246/28581556-257a6b4c-7180-11e7-8032-e3924ae2af47.png)

**Expected Behaviour**
We should get a deployment on each PR.

**Steps to reproduce it**

https://github.com/fossasia/meilix-generator/pull/87

**Comment by meets2tarun** [comment]
We need to rectify this issue soon
![screenshot from 2017-08-27 16-46-46](https://user-images.githubusercontent.com/15213246/29749397-dbce7628-8b47-11e7-929e-372091cfda1d.png)

**Comment by ParthS007** [comment]
@meets2tarun  is this issue still exists? If yes please tell me how to start with it.

**Comment by meets2tarun** [comment]
Yes, @ParthS007 the issue still persists, you can go through https://github.com/fossasia/meilix-generator/blob/master/app.json and look for the code to deployment.
@tabesin

**Comment by ParthS007** [comment]
@meets2tarun Is this issue still persists.Sorry for inactivity

**Comment by xeon-zolt** [comment]
> @meets2tarun Is this issue still persists.Sorry for inactivity

yes

**Comment by ParthS007** [comment]
thanks @xeon-zolt 👍 . Will let you know about the progress

**Comment by abishekvashok** [comment]
@meets2tarun on it.

**Comment by abishekvashok** [comment]
@meets2tarun @xeon-zolt where are the env. variables namely, EMAIL< KEY AND TRAVIS_TAG assigned?

**Comment by xeon-zolt** [comment]
@abishekvashok these three variables are defined in config vars in the setting of Heroku dashboard.

**Comment by abishekvashok** [comment]
The error states that they are missing, is it still the case?

**Comment by xeon-zolt** [comment]
cause the Heroku CI is not enabled and to enable it we require to add a credit card

**Comment by ParthS007** [comment]
@xeon-zolt Can we change the account?

**Comment by xeon-zolt** [comment]
@ParthS007 yes you can we only need this so that pull request can be tested easily
![screenshot-2017-12-13 meilix-generator settings Heroku 1](https://user-images.githubusercontent.com/4529442/33958814-9befa936-e06b-11e7-88df-34eacde306b8.png)

these are the three env variables two of them are empty and KEY contains the auth token for Travis CI API

**Comment by ParthS007** [comment]
@xeon-zolt can Transfer of ownership resolve the issue?

**Comment by 0x48piraj** [comment]
I'm getting started, I'll give an update if any fix is found.

**Comment by xeon-zolt** [comment]
@ParthS007 i don't think ownership transfer can help

**Comment by tabesin** [comment]
Is this resolved?
